src/appmixer/hibob/employees/DeletedEmployee/DeletedEmployee.js

```diff
@@ -0,0 +1,75 @@
+'use strict';
+const Promise = require('bluebird');
+
+/**
+ * Process employees to find newly deleted.
+ * @param {Array} currentEmployees
+ * @param {Array} deletedEmployees
+ * @param {Object} employeeId
+ */
+
+
+function processEmployees(currentEmployees, deletedEmployees, employeeId) {
+    if (!currentEmployees.includes(employeeId)) {
+        deletedEmployees.push(employeeId);
+    }
+}
+
+/**
+ * Component which triggers whenever an employee is deleted.
+ */
+class DeletedEmployee {
```

Comment:
Is there a reason for defining a class and later exporting it? Can it be simplified instead like this?

```suggestion
module.exports = {
```

Reply:
The reason for this is simply that it was required for our logging framework, which I removed from the components as it would not be relevant to you